File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/Result.java
package com.sdkwork.ai.model;

public class Result {
    private Integer index;
    private Double score;
    private String document;

    public Integer getIndex() {
        return this.index;
    }
    
    public void setIndex(Integer index) {
        this.index = index;
    }

    public Double getScore() {
        return this.score;
    }
    
    public void setScore(Double score) {
        this.score = score;
    }

    public String getDocument() {
        return this.document;
    }
    
    public void setDocument(String document) {
        this.document = document;
    }
}

File: sdkwork-ai-sdk-csharp/Models/Result.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class Result
    {
        public int? Index { get; set; }
        public double? Score { get; set; }
        public string? Document { get; set; }
    }
}
